### DISCUSSION TITLE
No audio after installing

### DISCUSSION THREAD

**Discussion by rAf19** [mrlt8/docker-wyze-bridge#816] [Q&A] [answered]
Installed in my DS920+ in Container Manager and even though installation is successful, I cannot hear any audio when I try to reproduce the RTSP URL in VLC.  Am I doing anything wrong?  How can I get the audio working?

**Comment by mrlt8** [comment, marked as answer]
Audio is not enabled by default, so you'll need to enable it in your docker environment with `ENABLE_AUDIO=True`
